internal/cli/alpha/manifestgen/manifestgen_test.go

```diff
@@ -10,16 +10,59 @@ import (
 	"fmt"
```

Comment:
It would be great to have tests also for Type and InterfaceGroup. If that too much work, please add TODO comment - but I believe it would be a copy of the Attribute test 🙂